staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -163,8 +163,10 @@ func RecordTransformation(transformationType, transformerPrefix string, start ti
 }
 
 // RecordCacheMiss records a miss on Key Encryption Key(KEK) - call to KMS was required to decrypt KEK.
-func RecordCacheMiss() {
-	envelopeTransformationCacheMissTotal.Inc()
+func RecordCacheMiss(cacheEnabled bool) {
```

Comment:
this parameter is a bit strange ... can we just avoid calling this if the cache was not enabled?

Reply:
Done.